### ISSUE TITLE
[BUG] Confirmation dialog size too big for extension popup  

### ISSUE THREAD

**Issue by mquasar** [getAlby/lightning-browser-extension#2378]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When deleting an allowance from the extension popup, confirmation dialog extends beyond UI boundaries.

### Screenshots [optional]

![image](https://user-images.githubusercontent.com/7760/235035451-9ca55b99-a3dd-456d-a9ef-c985e74ad95f.png)

![image](https://user-images.githubusercontent.com/349299/235036413-b3953c28-fb05-403c-8db0-bbf6fca30126.png)

### Steps To Reproduce

1. Visit a connected website, for example, [GetAbly](https://getalby.com/login)
2. Open extension from extension toolbar icon
3. On the allowance section, click on the settings icon
4. Click on delete

### Expected behavior

The confirmation dialog should appear within the boundaries of the extension. 
Another approach could be opening the allowance settings in a new tab with more room, just like the websites menu item, for instance.

### Alby information

- Alby 1.28.1 version for Firefox

### Device information

- Desktop

### Are you working on this?

None

**Comment by mquasar** [comment]
![image](https://user-images.githubusercontent.com/349299/235563066-8d5ec18c-0f94-44d2-a5a9-6f39e18813e8.png)

I tried this on Brave and the behavior is slightly different. The dialog shows on top of the extension and decentralized.

**Comment by fczuardi** [comment]
Looking at the code, this dialog is a native one from the browser, a [window.confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm), so the behavior will vary between browser implementations...

**Comment by im-adithya** [comment]
> this dialog is a native one from the browser, a [window.confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm)

Yes, this although can be replaced by a custom popup, but is it worth the effort? // @stackingsaunter @bumi

**Comment by stackingsaunter** [comment]
I am currently redesigning this part of the extension and I am going to propose not allowing to delete the website connection in the popup (since it deletes the whole connection, not allowence). Then the problem will disappear
